Doc: Add a known issue for the SDC (#10992)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

.. rst-class:: v2-3-0

> DRGN-18424: Rare assert when stopping the Periodic Advertiser role when another instance of the Periodic Advertiser role is still running.
>   When a Periodic Advertiser instance is being stopped while other Periodic Advertiser instances are still running, the controller might assert in rare cases.

Reviewer: ```suggestion
DRGN-18424: Rare assert when stopping the Periodic Advertiser role
  In rare cases, when a Periodic Advertiser instance is being stopped while other Periodic Advertiser instances are still running, the controller can assert.
```

.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0

> DRGN-18424: Rare assert when stopping the Periodic Sync role when another instance of the Periodic Sync role is still running.
>   When a Periodic Sync instance is being stopped while other Periodic Sync instances are still running, the controller might assert in rare cases.

Reviewer: ```suggestion
DRGN-18424: Rare assert when stopping the Periodic Sync role
  In rare cases, when a Periodic Sync instance is being stopped while other Periodic Sync instances are still running, the controller can assert.
```
